src/Configure.jl

```diff
@@ -246,7 +246,12 @@ function import_module_on_workers(procs, filename::String, options::Options, ver
     else
         @info "Importing SymbolicRegression on workers as well as extensions $(join(relevant_extensions, ',' * ' '))."
     end
-    @everywhere procs Base.MainInclude.eval($expr)
+    if VERSION < v"1.11.0-alpha1"
+        @everywhere procs Base.MainInclude.eval($expr)
+    else
+        # See https://github.com/JuliaLang/julia/issues/54057
+        @everywhere procs Core.eval(Base.MainInclude, $expr)
```

Comment:
```suggestion
        @everywhere procs Core.eval(Core.Main, $expr)
```

Reply:
And this works unconditionally, so you don't need the version check

Reply:
Cool, seems to work, thanks!

Btw, is this the right way to load extension code on dynamically allocated processes? I basically just:

1. Define a list of modules that trigger extensions, 
2. Check which ones are loaded on the head worker,
3. Load those modules on the workers

There are other difficulties in that the workers don't pick up the `v1.11` project and just activate directly in the current project, which might mean they don't have certain modules available. But I suppose that is unavoidable.

Reply:
Don't know. The Distributed stuff predates our current dynamic loading implementation, so there isn't an intended way to do it per-se. If it works, I think it's fine.

Reply:
Ok thanks, will do this for now